packages/jsii-pacmak/test/build-test.sh

```diff
@@ -46,7 +46,7 @@ else
     # Hello Windows!
     . ${venv}/Scripts/activate
 fi
-${PYTHON} -m pip install --upgrade pip~=22.1 twine~=4.0
+${PYTHON} -m pip install --upgrade pip~=23.3 twine~=4.0 setuptools~=67.3.2
```

Comment:
The venv isn't actually used to run pacmak for some reason. So this change doesn't have any effect on the failure in question. I could revert it, but feels better to also require an explicit version here.